### PR TITLE
fix factor in adjoint gradient from symmetry

### PR DESCRIPTION
**Pull request by mochen4**
Fix the factor when using symmetry in simulation

### CODE REVIEW

File: src/vec.cpp [hunk 1]
```diff
@@ -1217,6 +1217,15 @@ int symmetry::multiplicity() const {
     return g;
 }
 
+int symmetry::multiplicity(vec &x) const {
+  int m = multiplicity();
+  for (int n=1; n<m; ++n){
+    if (transform(x,n) == x)
```
Comment (stevengj) [inline src/vec.cpp:1223]: I'm worried that this test might fail due to roundoff errors.  Maybe it should apply to an `ivec` (integer coordinates) instead?

Comment (mochen4) [reply src/vec.cpp:1223]: Ok I will fix it

Comment (mochen4) [reply src/vec.cpp:1223]: I switched to ivec, and the code worked for me.

**Comment by smartalecH** [comment]
Maybe add a test to python/tests/adjoint_solver.py?

We actually don't have *any* far field adjoint tests yet. 

I would also make sure you're based off master.

**Pull request merged**